plaso/storage/sqlite/merge_reader.py

```diff
@@ -159,8 +159,8 @@ def _AddEventData(self, event_data, serialized_data=None):
     self._storage_writer.AddEventData(
         event_data, serialized_data=serialized_data)
 
-    identifier = event_data.GetIdentifier()
-    self._event_data_identifier_mappings[lookup_key] = identifier
+    post_write_identifier = event_data.GetIdentifier()
```

Comment:
maybe last_write_identifier ?

Reply:
I prefer 'post' as the prefix here, as this an identifier for the same event data, just after its been written to the store. 'last' to me suggests that it might be from a different object.

Reply:
but it contains the identifier of the last written event data correct? not the identifier AFTER (or post) the last written event data?

Reply:
Done